songs/api/serializers.py

```diff
@@ -0,0 +1,61 @@
+from django.db import models
+from rest_framework import serializers
+
+from songs.models import Song, Comment, Tag, Album
+
+
+class TagSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Tag
+        fields = ['content']
+
+    def validate_content(self, value):
```

Comment:
do we really need it? Cant we specify max-length in model or serializer field?

Reply:
Yes we can do it that way too. I'll remove this method